From b9586667d9fc012cc48f68a79eb264a62c926704 Mon Sep 17 00:00:00 2001
From: celadon <celadon@intel.com>
Date: Tue, 12 Sep 2023 11:20:31 +0000
Subject: [PATCH] Modification of IVI kernel

Modified IVI kernel to work for RPL NUC
- Modified dsp_driver flag from 4 to 1 at BoardConfig.mk

Signed-off-by: Yerriswamy <yerriswamy.kt@intel.com>
---
 groups/kernel/BoardConfig.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/kernel/BoardConfig.mk b/groups/kernel/BoardConfig.mk
index bcfa1fa..58a40a0 100644
--- a/groups/kernel/BoardConfig.mk
+++ b/groups/kernel/BoardConfig.mk
@@ -81,7 +81,7 @@ BOARD_KERNEL_CMDLINE += \
 endif
 
 BOARD_KERNEL_CMDLINE += \
-      snd_intel_dspcfg.dsp_driver=4 \
+      snd_intel_dspcfg.dsp_driver=1 \
       snd_soc_core.dyndbg==pmf \
       snd_hda_core.dyndbg==pmf \
       snd_hda_ext_core.dyndbg==pmf \
-- 
2.39.2

